osquery/remote/enroll/tests/enroll_tests.cpp

```diff
@@ -57,6 +57,10 @@ TEST_F(EnrollTests, test_enroll_key_retrieval) {
   // Turn the enrollment features back on and expect a key.
   FLAGS_disable_enrollment = false;
   EXPECT_EQ(getNodeKey("test_simple"), "fetched_a_node_key");
+
+  // DECLARE_string(FLAGS_enroll_secret_env);
+  // FLAGS_enroll_secret_env = "simple_enroll";
+  // EXPECT_EQ(getEnrollSecret(), "simple_enroll");
```

Comment:
We can punt on the test for now as it will be added in the other staged PR, but let's remove the dead commented code.